docs/adr/001-aligning-engines-field-with-ci.md

```diff
@@ -0,0 +1,69 @@
+# ADR 001: Aligning `engines` Field in `package.json` with CI Version Support
+
+## Status
+Proposed
+
+## Submitters
+- Ulises Gascón (@UlisesGascon)
+
+## Decision Owners
+- Express TC (@expressjs/express-tc)
+
+## Context
+The issue of maintaining consistency between the `engines` field in `package.json` and the versions tested in CI has been raised. Several packages have dropped support for Node.js versions prior to v18, but this change has not been consistently applied across all packages.
+
+**Why do we need this decision?**  
+Aligning the `engines` field with CI ensures that we explicitly define the minimum Node.js version supported for each release. This minimizes the risk of unintentional support for older versions that are not tested in CI, thus reducing maintenance overhead and unexpected bug reports.
+
+**What problem does it solve or avoid?**  
+This decision will help avoid introducing breaking changes unintentionally, streamline user expectations, and ensure that end users are aware of the minimum Node.js version they should use.
+
+**Are there any existing issues/discussions/pull requests related to this?**  
+- [Discussion: Using engines in the package.json #286](https://github.com/expressjs/discussions/issues/286)
+- [Missing engines Field Update for Node.js Version Support in v2.0.0 pillarjs/finalhandler#64](https://github.com/pillarjs/finalhandler/issues/64)
+
+## Decision
+We will establish a policy that requires the `engines` field in `package.json` to match the Node.js versions defined in our CI configuration. This policy will apply to all packages within the Express ecosystem.
+
+**What will be done?**  
+- Update the `engines` field in `package.json` for each package to reflect the minimum Node.js version supported by the corresponding CI configuration.
+
```

Comment:
```suggestion
- Include the `engines` fields in all the projects, so it will clarify what is supported for the final user

```

suggested by @ctcpip on the last TC Meeting